### PR TITLE
Add specs to verify current publication date behavior

### PR DESCRIPTION
**Pull request by astronaut-wannabe**
These specs are prepwork for [Issue #431][0]

They verify the current behavior of the `admin/article` form when it
comes to setting a publication date for an article and when it comes
to clicking the `Publish NOW!` button

### TODO

- [x] some unrelated specs are failing on `master`, fix those in a separate PR first
  - fixed in #433 and #434 
- [x] ~this is the first feature spec that requires `:js` to be on, I need to figure out how to configure Travis CI to run these specs in chrome headless (currently they run only locally in firefox with GUI)~
  will be fixed in #435

[0]:https://github.com/crimethinc/website/issues/431

### CODE REVIEW

**Comment by veganstraightedge** [comment]
Merged #435

**Comment by astronaut-wannabe** [comment]
@veganstraightedge thanks. this PR is done too now. let me know if you need an explanation on any of the changes

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ ISSUES.md
 /vendor/gems
 /coverage
 .idea/
+TAGS
```
Comment (veganstraightedge) [inline .gitignore]: What is this being ignored?

Comment (astronaut-wannabe) [inline .gitignore]: it is the default [ctags][0] file for emacs. `tags` (all lowercase) is the default ctags file for vim

[0]:https://blog.sensible.io/2014/05/09/supercharge-your-vim-into-ide-with-ctags.html

**Pull request merged**